pet-store-vanilla-js/utils.js

```diff
@@ -0,0 +1,22 @@
+export function isNullOrWhitespace(value) {
+    if (value == null || value.trim().length == 0) {
+        return true;
+    }
+
+    return false;
+};
+
+export function setDatePrototypeFormating() {
+    Date.prototype.getFormattedDate = function () {
```

Comment:
* Don't overwrite built in prototypes.
* Don't hardcode month names like this. There is a browser API that can help you. Google it.

Reply:
The prototype override has been removed.
I created function `formatDate` in `utils.js` that uses the browser api for formatting